src/pretix/control/forms/item.py

```diff
@@ -141,6 +141,38 @@ def __init__(self, *args, **kwargs):
             required=False
         )
 
+        self.fields['quota_option'] = forms.ChoiceField(
+            label=_("Quota options"),
+            widget=forms.Select,
+            choices=(
+                (0, 'None'),
+                (1, "Add this to existing quota"),
+                (2, "Add this to new quota")
```

Comment:
First, the choices should also be translatable, e.g. ``_("Add this to existing quota").
Second, I'd find the following texts more clear:
* Do not add to a quota now
* Add product to an existing quota
* Create a new quota for this product

Also, I'd recommend you to use strings like ``none``, ``existing`` and ``new`` instead of 0,1,2. it makes the code easier to read